### DISCUSSION TITLE
/next-sitemap.config.js

### DISCUSSION THREAD

**Discussion by RealEstateSaga** [community/community#119745] [Programming Help]
### Select Topic Area

Question

### Body

Greetings, can I delete this: /next-sitemap.config.js
Will this only hav an impact on search.
If I delete this it will have no impact on how the site operates, only how search engines work: /next-sitemap.config.js

Thank you!

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @RealEstateSaga !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Code Search and Navigation category is solely related to conversations around the GitHub product CS&N. This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
